zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/authorization/UpdateRoleMultiPartitionTest.java

```diff
@@ -136,13 +135,13 @@ public void distributionShouldNotOvertakeOtherCommandsInSameQueue() {
 
   private void interceptUserCreateForPartition(final int partitionId) {
     final var hasInterceptedPartition = new AtomicBoolean(false);
-    ENGINE.interceptInterPartitionCommands(
+    engine.interceptInterPartitionCommands(
         (receiverPartitionId, valueType, intent, recordKey, command) -> {
           if (hasInterceptedPartition.get()) {
             return true;
           }
           hasInterceptedPartition.set(true);
-          return !(receiverPartitionId == partitionId && intent == RoleIntent.CREATE);
```

Comment:
🔧  I'd get rid of the user creation in this test alltogether. There's no need to create a user. Just create a role and intercept the `RoleIntent.CREATE` acknowledgedgements, which you were already doing.